lgn-provers/src/provers/v1/preprocessing/euclid_prover.rs

```diff
@@ -54,16 +54,16 @@ impl EuclidProver
         skip_store: bool,
     ) -> anyhow::Result<Self>
     {
-        debug!("Creating preprocessing prover");
-        let params = ParamsLoader::prepare_bincode(
+        let params = ParamsLoader::prepare_raw(
             url,
             dir,
             file,
             checksum_expected_local_path,
             skip_checksum,
             skip_store,
         )?;
-        debug!("Preprocessing prover created");
```

Comment:
These logs are already emitted by `create_prover` functions. Removed the duplicates.